pages/blog/integrating-mailchimp-with-your-nodejs-app.mdx

```diff
@@ -0,0 +1,350 @@
+---
+title: "Integrating MailChimp with your Node.js App"
+date: "2021-06-01"
+author: "Yashovardhan Agrawal"
+pullQuote: "A step by step guide of integrating your MailChimp Audiences with your Node.js app"
+image: "integrating-mailchimp-with-your-nodejs-app/mc4.png"
+tags: [connections, engineering]
+---
+
+# Integrating MailChimp with your Node.js App
+
+If you're reading this blog right now, the chances are that you're finding your way to add a MailChimp integration into your Node.js application. Don't worry; we have you covered! This is a step by step guide that will help you build a sample Node.js backend for your webpage and help you integrate it with the MailChimp API.
+This tutorial will discuss all the different things you need to look for within MailChimp with references to their excellent API documentation. We'll also talk about an alternative method using Grouparoo, which will do all the heavy lifting for you!
+
+## Getting started
+
+To get started, you'll be needing the following things prepared:
+
+- A MailChimp Account
+- Node.js & npm installed on your machine
+- A basic frontend application to send requests to your Node.js application.
+
+This guide will help you make a sample Node.js application with Express to send and receive API requests and responses. Although a basic knowledge of Node.js would be helpful, you can easily follow the steps given and get started with your application.
```

Comment:
When paired with the previous comment, I'm starting to feel like you're asking the reader to do a lot without a direct reference.

If we're going super bare-bones here, which it looks like you are, I'm wondering if you should provide an example repo. You could include both the front and the back end. And if you build it out incrementally, you could point to specific points in time using git commit hashes on GitHub. And in doing so, you're getting started section becomes way simpler. You point to a README this example repo that asks you to clone, install dependencies, and then provides two commands — one to run the front end, one to run the back end.

Reply:
I feel the w3schools sample frontend will be helpful here.

I'm rather on the fence on providing the whole code as a GitHub repository since people might tend to copy that instead of reading through this and preferring grouparoo as a much better option.